cdk-infra/lib/constructs/auto-builder-env-construct.ts

```diff
@@ -0,0 +1,51 @@
+import { IQueue } from 'aws-cdk-lib/aws-sqs';
+import { StringParameter } from 'aws-cdk-lib/aws-ssm';
+import { Construct } from 'constructs';
+
+interface AutoBuilderEnvConstructProps {
+  jobsQueue: IQueue;
+  jobUpdatesQueue: IQueue;
+}
+export class AutoBuilderEnvConstruct extends Construct {
+  readonly environment: Record<string, string>;
+
+  constructor(scope: Construct, id: string, props: AutoBuilderEnvConstructProps) {
+    super(scope, id);
+
+    const dbName = StringParameter.valueFromLookup(this, '/env/dev/docs/worker_pool/atlas/dbname');
+    const dbUsername = StringParameter.valueFromLookup(this, '/env/dev/docs/worker_pool/atlas/username');
+    const dbHost = StringParameter.valueFromLookup(this, '/env/dev/docs/worker_pool/atlas/host');
+    const dbPassword = StringParameter.valueFromLookup(this, '/env/dev/docs/worker_pool/atlas/password');
+    const slackSecret = StringParameter.valueFromLookup(this, '/env/dev/docs/worker_pool/slack/webhook/secret');
```

Comment:
Should we be using `fromSecureStringParameterAttributes` here for secure strings, or is it okay to use `valueFromLookup`? I noticed `fromSecureStringParameterAttributes` was used elsewhere for Fastly tokens, so I wanted to get an understanding of this was needed here

Reply:
We should, but `SecureStringParameters` from SSM can't be used in CloudFormation (which is what is used under the hood). I checked, and we currently grab the strings as non-secrets when using the Serverless stack as well. I talked to Cassidy about this, and our temporary workaround is to add the `cdk.context.json` to the gitignore. 

In the future, we could use the secrets manager to access secrets.